libs/blocks/author-header/author-header.js

```diff
@@ -0,0 +1,8 @@
+export default function init(blockEl) {
+  const title = blockEl.querySelector('div:nth-child(1)');
+  if (title) { title.classList.add('author-header-title'); }
+  const img = blockEl.querySelector('div:nth-child(2)');
+  if (img) { img.classList.add('author-header-img'); }
+  const bio = blockEl.querySelector('div:nth-child(3)');
+  if (bio) { bio.classList.add('author-header-bio'); }
```

Comment:
Are we not adding social links as well or is this deliberately not being done?

Reply:
@vhargrave  Not sure about this one actually. I also thought of adding social links but the older block [https://github.com/adobe/blog/blob/main/blocks/author-header/author-header.js](url) not include the social links. 

@auniverseaway Should we add the social links on the `author-header` block?

Reply:
I will add the social links, it's better to have them added.